debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -391,6 +401,10 @@ public String getDatabaseName() {
         return databaseName;
     }
 
+    public String getInstanceName() {
```

Comment:
I'm still struggling with this; where is this used? Only in your custom connection factory, @mallikankati?